ports/icon_hal/icon_hal.cmake

```diff
@@ -0,0 +1,32 @@
+# This CMakeLists is a template for new ports. It provides the minimal
+# configuration for building, but nothing more.
+
+# Warn that the template port only builds. It will not create usable libraries.
+message( STATUS "Configurint AWS to use the Icon HAL")
+
+# Template platform sources. A network implementation (not listed below) is also needed.
+set( PLATFORM_SOURCES
+     ${PORTS_DIRECTORY}/${IOT_PLATFORM_NAME}/src/iot_clock_${IOT_PLATFORM_NAME}.c
+     ${PORTS_DIRECTORY}/${IOT_PLATFORM_NAME}/src/iot_threads_${IOT_PLATFORM_NAME}.c
+     ${PORTS_DIRECTORY}/${IOT_PLATFORM_NAME}/src/iot_port_mem_helper.c
+     ${PORTS_DIRECTORY}/${IOT_PLATFORM_NAME}/src/iot_network_mbedtls.c
+     # ${PORTS_DIRECTORY}/common/src/iot_network_metrics.c
```

Comment:
Should these be commented out or just deleted?

Reply:
Deleted.